hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/SstFilteringService.java

```diff
@@ -231,4 +241,10 @@ public AtomicLong getSnapshotFilteredCount() {
   public BootstrapStateHandler.Lock getBootstrapStateLock() {
     return lock;
   }
+
+  @Override
```

Comment:
nit: stop is in opposite order of start.

```suggestion
  @Override
  public void shutdown() {
    super.shutdown();
    running.set(false);
  }
```

Reply:
The order is correct. when running is set to false the loop will break for processing the next snapshot. It would be better if we do it this way.

Reply:
Is this needed?  As I mentioned above, the keymanager is already stopping the service.